practice-two/src/components/ShoppingCart.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+
+// Components
+import Button from './common/Button/Button';
+import Popover from './common/Popover';
+import Typography from './common/Typography';
+
+// Data
+import { ICON } from '../constants/data';
+
+/**
+ * ShoppingCart Component
+ *
+ * A popover component for displaying cart products.
```

Comment:
Wrong! It's not a popover. It's sidebar.  and it's should not relative to 'Popover' component

Reply:
Done, Updated by Slider components